Healthy/Classes/Extensions/UIButton+Style.swift

```diff
@@ -5,17 +5,35 @@ import UIKit
 extension UIButton {
     enum ButtonStyle {
         case primary
-        case secondary
-        case disable
-
-        var buttonColor: UIColor {
+        case plainGold
+        
+        // TODO: [HL-4] Add global Color
+        var buttonBackgroundColor: UIColor? {
```

Comment:
```suggestion
        private var buttonBackgroundColor: UIColor? {
```

Reply:
I cannot make these variables private because the function in which these variables are used is internal, so these variables must be made internal as well.

Reply:
Let's make it `fileprivate` instead. This applies to the all other computed properties.